cmd/gitquery/query_base.go

```diff
@@ -17,7 +17,7 @@ import (
 type cmdQueryBase struct {
 	cmd
 
-	Path []string `short:"p" long:"path" description:"Path where the git repository is located"`
+	Path []string `short:"p" long:"path" description:"Path where the git repository is located, can be used several times"`
```

Comment:
it should be "where the git repositories are located", right?

Reply:
Changed as now it supports multiple repos in a directory.